core/stat/base/sliding_window_metric.go

```diff
@@ -101,6 +101,10 @@ func (m *SlidingWindowMetric) GetQPS(event base.MetricEvent) float64 {
 	return m.getQPSWithTime(util.CurrentTimeMillis(), event)
 }
 
+func (m *SlidingWindowMetric) GetPreviousQPS(event base.MetricEvent) float64 {
+	return m.getQPSWithTime(util.CurrentTimeMillis()-uint64(m.bucketLengthInMs), event)
```

Comment:
The initial design of the method is for getting the metric of last second (i.e. elapsed, "stable" second). Here we got the latest 1 sec (with current bucket, i.e. unstable bucket). We may discuss it later.